src/vidpf.rs

```diff
@@ -452,6 +489,13 @@ struct VidpfPrgOutput {
     right_control_bit: Choice,
 }
 
+/// Clients use this structure to send reports to each of the Servers.
+pub struct ClientReport<V: VidpfValue> {
```

Comment:
This probably doesn't need to be public, I figure it will just be used for tests here. Higher-level protocols such as DAP will be responsible for bundling the public share, input share, and nonce together in their own way, and that will include additional fields from Mastic in the shares as well.